src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -420,4 +463,43 @@ public void changeItemStatus(String itemStatus) {
   public void changeDeclaredLostDateTime(DateTime dateTime) {
     write(representation, LoanProperties.DECLARED_LOST_DATE, dateTime);
   }
+
+  private class Policies {
+    private LoanPolicy loanPolicy;
+    private OverdueFinePolicy overdueFinePolicy;
+    private LostItemPolicy lostItemPolicy;
```

Comment:
Can these be made final?

Reply:
these fields can't be final as they can be changed

Reply:
And they are changed in situ rather than a new policies class instance being created?

Reply:
Changed